.vib/airflow/goss/airflow.yaml

```diff
@@ -0,0 +1,20 @@
+command:
+  check-scheduler:
+    timeout: 30000
+    exec: airflow users list
+    exit-status: 0
+    stdout:
+      - No data found
+  check-so-files:
+    # Check that all .so files in site-packages are executable
+    exec: find /opt/bitnami/airflow/venv/lib/python3.9/site-packages/ -type f -name '*.so.*' ! -executable | wc -l
+    exit-status: 0
+    stdout:
+      - 0
```

Comment:
I don't think this adds much value. Those files are third party packages...

Reply:
Instead, you could list the `venv` for submodules that should've been installed, such as `celery`, `docker`, `cncf.kubernetes` or `hashicorp`